app/scripts/models/account.js

```diff
@@ -96,7 +96,8 @@ define([
             self.set('verified', verified);
           }, function () {
             // Ignore errors; we'll just fetch again when needed
-          });
+            // TODO Remove once https://github.com/eslint/eslint/issues/3279 gets fixed
```

Comment:
Bithound Driven Development?

@vladikoff Not update what, ESLint?
I would probably lean towards keeping it updated since it's useful and clearly has caught a few issues w/ indentation (admittedly a pretty weak rule and is purely stylistic). But there is definitely more churn than I expected here. I thought it'd only be 3-4 indenting issues with the empty blocks.

Looks like we've disabled ESLint `indent` rule in 3 files, on 6 lines, and disabled ESLint completely for 3 blocks.